applications/ehr_query_llm/lmm/run_lmm.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+# Array to hold PIDs of background processes
+declare -a bg_pids
+
+set -x 
+
+# Function to clean up and kill the background processes
+cleanup() {
+    echo "Terminating Llama.cpp server process..."
+    # Kill the /bin/server process.
+    pkill -f "/opt/nvidia/holoscan/llama.cpp/build/bin/server"
+
+    # Kill all background Python processes
+    for pid in "${bg_pids[@]}"; do
+        if kill -0 $pid > /dev/null 2>&1; then
+            kill $pid
+        fi
+    done
+
+    exit 0
+}
+
+set_python_path() {
+    # Set the python path to the holoscan_sdk_install path
+    local holoscan_sdk_install=$(grep -Po '^holoscan_DIR:PATH=\K[^ ]*' build/CMakeCache.txt)
+    local holohub_build_dir="${SCRIPT_DIR}/build"
+    local environment="export PYTHONPATH=${holoscan_sdk_install}/../../../python/lib:${holohub_build_dir}/python/lib:${SCRIPT_DIR}"
```

Comment:
If `build/CMakeCache.txt` is not available this can cause cascading failures in environment availability, @cdinea I think you reported this in the past where `holoscan` was not found. Could you please add a hard check to verify `holoscan` is available after setting PYTHONPATH? Something like:

```suggestion
    local holoscan_sdk_install=$(grep -Po '^holoscan_DIR:PATH=\K[^ ]*' build/CMakeCache.txt)
    local holohub_build_dir="${SCRIPT_DIR}/build"
    local environment="export PYTHONPATH=${holoscan_sdk_install}/../../../python/lib:${holohub_build_dir}/python/lib:${SCRIPT_DIR}"
    # Check that Holoscan is available in PYTHONPATH
    python -c "import holoscan" || ("Failed to import Holoscan module" && exit 1)
```

Reply:
Resolved by latest changes 👍